habitat-lab/habitat/datasets/rearrange/run_episode_generator.py

```diff
@@ -4,12 +4,17 @@
 # This source code is licensed under the MIT license found in the
 # LICENSE file in the root directory of this source tree.
 
+import ctypes
 import os
 import os.path as osp
 import random
+import sys
 from dataclasses import dataclass, field
 from typing import TYPE_CHECKING, Any, List
 
+flags = sys.getdlopenflags()
```

Comment:
Any reason we don't just have this happen as part of the habitat root import?

Reply:
Is this still needed after https://github.com/facebookresearch/habitat-sim/pull/2105? It's a nasty hack, best avoided if possible.

Or is this fixing some other issue than missing plugins?

Reply:
Good question, I'll check that this is still necessary.

Reply:
Seems unnecessary now. Removed. :+1: